dev/wlp-gradle/subprojects/tasks.gradle

```diff
@@ -360,7 +360,8 @@ test {
   }
 
   timeout = Duration.ofMinutes(15)
-  ignoreFailures = Boolean.valueOf(rootProject.ext.userProps.getProperty("gradle.test.ignoreFailures", "false"))
+  //Set ignore failures using -Dgradle.test.ignoreFailures=true
+  ignoreFailures = Boolean.valueOf(System.getProperty("gradle.test.ignoreFailures", "false"))
```

Comment:
this looks fine, but lets be sure to test that it still works on an RTC build when there is a unit test failure